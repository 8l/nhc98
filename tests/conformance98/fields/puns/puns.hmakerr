======	Errors when renaming:
Named field puns are not Haskell'98 - used at 3:26
Named field puns are not Haskell'98 - used at 3:13
Named field puns are not Haskell'98 - used at 3:11
