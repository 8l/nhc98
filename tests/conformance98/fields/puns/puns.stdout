Point 0 1
